src/ophys_etl/modules/trace_extraction/schemas.py

```diff
@@ -0,0 +1,65 @@
+from argschema import ArgSchema
+from argschema.schemas import DefaultSchema
+from argschema.fields import (LogLevel, String, Nested, Float,
+                              OutputDir, InputFile)
+
+from ophys_etl.schemas.fields import H5InputFile
+from ophys_etl.schemas import ExtractROISchema
+
+
+class MotionBorder(DefaultSchema):
+    # TODO: be really certain about how these relate to
```

Comment:
Has this been addressed?

Reply:
looks like that just came along for the ride from AllenSDK:
https://github.com/AllenInstitute/AllenSDK/blob/ea8cdc760210bec5ad01a3bd8fd586b6e2228080/allensdk/brain_observatory/ophys/trace_extraction/_schemas.py#L10-L14
I'll remove the comment.

Reply:
fixed